pkg/virt-handler/isolation/detector_test.go

```diff
@@ -155,3 +155,22 @@ var _ = Describe("getMemlockSize", func() {
 		Expect(int(bytes)).To(Equal(1264389000))
 	})
 })
+
+var _ = Describe("findIsolatedQemuProcess", func() {
+	It("should return the QEMU process", func() {
+		virtLauncherPid := 1
+		virtLauncherProc := ProcessStub{pid: virtLauncherPid, ppid: 0, binary: "virt-launcher"}
+		virtLauncherForkedProc := ProcessStub{pid: 26, ppid: 1, binary: "virt-launcher --no-fork true"}
+		libvirtdProc := ProcessStub{pid: 226, ppid: 26, binary: "libvirtd"}
+		qemuProc := ProcessStub{pid: 101, ppid: 1, binary: qemuProcessExecutable}
+		virtLauncherProcesses := convertToPsGoProcessList([]ProcessStub{
+			virtLauncherProc,
+			qemuProc,
+			virtLauncherForkedProc,
+			libvirtdProc,
+		})
+		qemuPsProcess := convertToPsGoProcess(&qemuProc)
```

Comment:
You cannot pass `&qemuProc` instead of `qemuPsProcess` ?

Reply:
Yep it works with `&qemuProc`, I removed `convertToPsGoProcess`.